src/main/java/dev/meyi/bn/modules/calc/BankCalculator.java

```diff
@@ -176,12 +169,7 @@ public static void evaluate(String message) {
   }
 
   public static synchronized void reset() {
-    if (startup) {
-      startup = false;
-      calculatedProfit = BazaarNotifier.config.bazaarProfit;
-    } else {
-      calculatedProfit = 0;
-    }
+    BazaarNotifier.config.bankModule.bazaarProfit = 0;
```

Comment:
just to verify, this works the same correct?

Reply:
Yes